src/sentry/models/projectownership.py

```diff
@@ -369,14 +369,27 @@ def _matching_ownership_rules(
         ownership: ProjectOwnership | ProjectCodeOwners,
         data: Mapping[str, Any],
     ) -> list[Rule]:
-        rules = []
-        if ownership.schema is not None:
-            munged_data = Matcher.munge_if_needed(data)
-            for rule in load_schema(ownership.schema):
-                if rule.test(data, munged_data):
-                    rules.append(rule)
-
-        return rules
+        if ownership.schema is None:
+            return []
+
+        # "projectownership" or "projectcodeowners"
+        ownership_type = type(ownership).__name__.lower()
+
+        munged_data = Matcher.munge_if_needed(data)
+        metrics.gauge(
```

Comment:
not an expert but would a distribution metric type be more appropriate here?

Reply:
oh yeah good catch, it definitely should be a distribution